CHANGELOG.rst

```diff
@@ -52,6 +52,64 @@ Detailed changes
 
 ...
 
+
+3.2.1 (2025-08-19)
+==================
+
+Upgrade procedure
+-----------------
+
+**💰 Ingenico Ogone migration**
```

Comment:
Not sure if we want to include this note here (as it is also mentioned in the 3.1.6 release)